src/pages/api/modules/rss/index.ts

```diff
@@ -70,6 +71,14 @@ export const Get = async (request: NextApiRequest, response: NextApiResponse) =>
   });
 };
 
+const createLink = (item: any) => {
+  if (item.link) {
+    return item.link;
+  }
+
+  return item.guid;
```

Comment:
What if there isn't this property either ? A mode widget-level error management system would be better to prevent further crashes.

Reply:
Then it will just return ``undefined``?
I agree, that we should have uniformed error handling.
I will check out https://nextjs.org/docs/advanced-features/error-handling#handling-client-errors

Reply:
Did the other PR fix this error ? If so we can merge